code/datums/craft/recipe.dm

```diff
@@ -67,6 +67,52 @@
 	var/datum/craft_step/CS = steps[step]
 	return CS.apply(I, user, target, src)
 
+/datum/craft_recipe/proc/build_batch(mob/living/user, amount)
+	if(steps.len > 1)
+		warning("Attempted to batch craft a multi-step recipe: [name]")
```

Comment:
do we really need this unnecessary logging ?

Reply:
User asks for fifty but gets one because some coder messed up a recipe? Yeah why should we log that

Reply:
mention that then

Reply:
```suggestion
		warning("A multi-step recipe has BATCH_CRAFT flag: [name]. It should not!")
```
Like that?

Reply:
yeah good enough